### DISCUSSION TITLE
【要望】「UIにぼかし効果を使用」「モーダルにぼかし効果を使用」をデフォルトでオフに

### DISCUSSION THREAD

**Discussion by nice714106tall** [misskey-dev/misskey#9783] [Ideas]
ぼかし効果についての要望です。
（以下村上さんの発言 https://misskey.io/notes/97fnvdwthd より引用）

>以下のリンクから
>「UIにぼかし効果を使用」
>「モーダルにぼかし効果を使用」
>をオフにすると大幅に軽くなります。
>
>また、Androidなどで絵文字パレットが点滅したり消えたりする問題もこれが原因なので、おかしくなっている方はオフにしてみて下さい。
>
>settings/general

（引用終わり）

新規ユーザーの初期値をオフにした方が、村上さんが言及しているようなデメリットを回避できて、新規ユーザーが増えるたびにアナウンスしたり問い合わせへの回答する手間も省けるのではないかと思いました。
また、描画崩れに遭遇していても回避方法へアクセスできずに、以前の私のように「そういうものなんだろう」「仕方ない」と思って我慢して不便なまま使用している人も少なからず居るのではないかと想像しています。
主観的な感想になりますが、ぼかし効果をオフにしてもUIの美しさが損なわれることは無いので、デフォルト値をオフにすることを要望として挙げさせていただきます。

**Comment by tamaina** [comment]
これは今そうなってないかしら
